src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -92,24 +75,25 @@ export const ProjectListItemCard: FC<ProjectListItemCardProps> = ({
       <div className={cardBody}>
         <CardMedia
           className={media}
-          image={projectImagePath}
+          image="/images/favicon-32x32.png" //Should be updated in next version.
           title="Project Image"
         />
         <CardContent className={cardContent}>
           <div className={leftContent}>
-            <Typography variant="h4">{name}</Typography>
+            <Typography variant="h4">{props.name?.value}</Typography>
             <Typography color="primary" variant="body2">
               <Box
                 component="span"
                 m={1}
                 display="inline"
                 color="text.secondary"
               >
-                {id}
+                {props.id}
               </Box>
-              {countryName}, {region}
+              {props.location?.value?.name.value},{' '}
+              {props.location?.value?.region.value?.name.value}
```

Comment:
This needs to be a bit smarter. Right now it's just showing a comma when there is no location. A helper function could be useful for this as well since we'll probably need to display locations in multiple areas

```tsx
const displayLocation = (location: Location /* From API, not global */) => '';
```